hoot-core-test/src/test/cpp/hoot/core/ops/TrivialOpTest.cpp

```diff
@@ -1,145 +0,0 @@
-/*
```

Comment:
This was a strange test to me... axed it.

Reply:
Yeah, good call.